frontend/src/components/EnnemyFly.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./style/EnnemyFly.scss";
+
+class EnnemyFly extends React.Component {
+  constructor(props) {
```

Comment:
This constructor doesn't serve any purpose (I think you have a message in your console about it)